kivymd/uix/selectioncontrol/selectioncontrol.kv

```diff
@@ -56,6 +56,8 @@
         pos_hint: {"center_x": .5, "center_y": .5}
         theme_text_color: "Custom"
         text_color:
+            self.theme_cls.surfaceContainerHighestColor if not root.parent else \
```

Comment:
I would solve this in the following way:

```python
<Thumb>
    [...]

    ThumbIcon:
        [...]

        text_color:
            [...]

            if root.parent else self.theme_cls.transparentColor
```

Reply:
Ok

Thanks for the recommendation. I pushed the modification.